README.md

```diff
@@ -1,87 +1,115 @@
 
-Carbon is a tool for generating a feed of people that can be loaded into Symplectic Elements. It is designed to be run as a container. This document contains general application information. Please refer to the [mitlib-tf-workloads-carbon](https://github.com/mitlibraries/mitlib-tf-workloads-carbon) for the deployment configuration.
+Carbon is a tool for loading data into [Symplectic Elements](https://support.symplectic.co.uk/support/solutions/articles/6000049890-symplectic-elements-quick-start-guide). Carbon retrieves records from the Data Warehouse, normalizes and writes the data to XML files, and uploads the XML files to the Elements FTP server. It is used to create and run the following feed types: 
+
+   * `people`: Provides data for the HR Feed.
+   * `articles`: Provides data for the Publications Feed.
+
+Please refer to the [mitlib-tf-workloads-carbon](https://github.com/mitlibraries/mitlib-tf-workloads-carbon) for the deployment configuration.
 
 ## Development
 
 * To install with dev dependencies: `make install`
 * To update dependencies: `make update`
-* To run unit tests: `make test`
 * To lint the repo: `make lint`
-* To run the app: `pipenv run carbon --help`
 
-The Data Warehouse runs on an older version of Oracle that necessitates the `thick` mode of python-oracledb, which requires the Oracle Instant Client Library (this app was developed with version 21.9.0.0.0). The test suite uses SQLite, so you can develop and test without connecting to the Data Warehouse.
+The Data Warehouse runs on an older version of Oracle that necessitates the `thick` mode of `python-oracledb`, which requires the Oracle Instant Client Library (this app was developed with version 21.9.0.0.0).
+
+### Running the test suite
+
+The test suite uses SQLite, so you can develop and test without connecting to the Data Warehouse.
 
-### With Docker
+1. Run `make test` to run unit tests.
 
-**Note:** As of this writing, the Apple M1 Macs cannot run Oracle Instant Client, so Docker is the only option for development on those machines.
+### Running the application on your local machine
 
-From the project folder:
+1. Export AWS credentials for the `Dev1` environment.
+2. Create a `.env` file at the root folder of the Carbon repo, and set the required environment variables described in [Required Env](#required-env).
 
-1. Export AWS credentials for the `dev1` environment.
+   **Note**: The host for the Data Warehouse is different when connecting from outside of AWS (which uses Cloudconnector). For assistance, reach out to the [Data Warehouse team](https://ist.mit.edu/warehouse).
 
-2. Run `make dependencies` to download the Oracle Instant Client from S3.
+3. If the run requires a connection to the Data Warehouse, connect to an approved VPN client. Otherwise, skip this step.
+4. Follow the steps relevant to the machine you are running:
+   * If you are on a machine that cannot run Oracle Instant Client, follow the steps outlined in [With Docker](#with-docker).
```

Comment:
I think this should include a note to skip `make publish-dev` since we just need to run `make dist-dev` to build the container locally right?

Reply:
Also agreed.  Will save for a more general comment, but I think this is a good example where some of these instructions deviate for purely local, docker testing vs testing the deployed container in Dev1.

Reply:
Added a sentence about skipping the step to publish the Docker container.